src/EnergyPlus/main.cc

```diff
@@ -63,6 +63,8 @@ using EnergyPlus::CommandLineInterface::ProcessArgs;
 int
 main( int argc, const char * argv[] )
 {
+	// the following line is only needed when debugging issues related to NaN in Visual Studio. See https://github.com/NREL/EnergyPlus/wiki/Debugging-Tips
+	// unsigned int fp_control_state = _controlfp( _EM_INEXACT, _MCW_EM );
```

Comment:
I don't think there is a way to handle that line in the build but it is already documented [here](https://github.com/NREL/EnergyPlus/wiki/Debugging-Tips#catching-a-nan) so it is not critical I just thought it would be more convenient for other developers.